lib/query.js

```diff
@@ -2951,11 +2951,14 @@ Query.prototype.distinct = function(field, conditions, callback) {
  *     // equivalent
  *     query.sort('field -test');
  *
+ *     // also possible is to use a array with array key-value pairs
+ *     query.sort([["field", "asc"]]);
```

Comment:
Nitpick: single quotes instead of double quotes.
```suggestion
 *     query.sort([['field', 'asc']]);
```

Reply:
fixed with 06b3c6e6445c1d89a3b88060f98788fe64ecd535